### DISCUSSION TITLE
Required fields are not validated when updating entries

### DISCUSSION THREAD

**Discussion by dsheiko** [directus/directus#5635] [Archive]
As an admin I have fields of type string/text with "allow NULL value" OFF.
When creating new entries the fields are validated for NULL values, but not when I update earlier created entries.

Expected behavior: it shall block saving and warn the user the value is empty whenever it creating or updating

Project details
Directus Version: 9.0.0-rc.64
Environment: production
OS: Ubuntu 18.04.2 LTS
Node: 11.15.1

**Comment by rijkvanzanten** [comment]
Make sure the `Cleared Value` option of your input is set to "Save as NULL" 🙂
